src/PresentationalComponents/PathwaysPanel/PathwaysPanel.js

```diff
@@ -54,7 +54,10 @@ const PathwaysPanel = () => {
         {isFetching ? (
           <Loading />
         ) : !isError && data.data?.length ? (
-          <Grid hasGutter md={4} sm={12}>
+          <Grid
+            md={data.data?.length === 1 ? 8 : data.data?.length === 2 ? 6 : 4}
```

Comment:
We can make it more readable in my opinion by adding parenthesis into the first else block.
```suggestion
            md={data.data?.length === 1 ? 8 : (data.data?.length === 2 ? 6 : 4)}
```

Reply:
Originally I tried this, but the linter gets angry